main.py

```diff
@@ -1,10 +1,47 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+    for nickname, data in players.items():
+        race_data = data["race"]
+        guild_data = data["guild"]
+        Player.objects.create(
+            nickname=nickname,
+            email=data["email"],
+            bio=data["bio"],
+            race=(
+                Race.objects.get(name=race_data["name"])
+                if Race.objects.filter(name=race_data["name"]).exists()
+                else
+                Race.objects.create(
+                    name=race_data["name"],
+                    description=race_data["description"]
+                )
```

Comment:
You can check this before player creation. And move the resulting race to a variable